Source/JSON/JSONReader.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  JSONReader.swift
+//  Matrioska
+//
+//  Created by Andreas Thenn on 12/01/2017.
+//  Copyright © 2017 runtastic. All rights reserved.
+//
+
+import Foundation
+
+/// A JSONObject type
+public typealias JSONObject = [String: Any]
+
+/// A JSONReader used to convert to JSONObject
+public final class JSONReader {
+    
+    /// Serializes from a given JSON Data into JSONObject
+    ///
+    /// - Parameter data: the Data object used in serialization
+    /// - Returns: an optional serialized JSONObject
+    /// - Throws: throws an error in case of failure or invalid JSON data
+    public class func jsonObject(from data: Data) throws -> JSONObject? {
+        let json = try JSONSerialization.jsonObject(with: data) as? JSONObject
+        
+        return json
+    }
+
+    /// Serializes from a given JSON file into JSONObject
+    ///
+    /// - Parameters:
+    ///   - jsonFilename: the file name
+    ///   - bundle: the bundle where the file is located
+    /// - Returns: an optional serialized JSONObject
+    /// - Throws: throws an error in case of failure or invalid JSON data
+    public class func jsonObject(from jsonFilename: String, bundle: Bundle = .main) throws -> JSONObject? {
```

Comment:
this shouldn't be part of Matrioska, I think the whole `JSONReader`shouldn't be part of Matrioska since it's just convenience functions for `JSONSerialization` and retrieving a file